ui/src/app/pages/analysis/runtime-panel.spec.tsx

```diff
@@ -887,4 +889,16 @@ describe('RuntimePanel', () => {
     await pickWorkerDiskSize(wrapper, 50);
     expect(getNextButton().prop('disabled')).toBeFalsy();
   });
+
+  it('should render disabled panel when creator billing disabled', async () => {
+    const wrapper = await component({workspace: {
+        ...workspaceStubs[0],
+        accessLevel: WorkspaceAccessLevel.WRITER,
+        billingStatus: BillingStatus.INACTIVE,
+        cdrVersionId: CdrVersionsStubVariables.DEFAULT_WORKSPACE_CDR_VERSION_ID
+      }});
+
+    const billingInactiveMessage = wrapper.find({'data-test-id': 'billing-inactive'});
+    expect(billingInactiveMessage.text()).toContain('disabled');
```

Comment:
I think two easy assertions to add would be
1. The `DisabledPanel` component exists
2. The `RuntimePanel` component does not exist

Reply:
`RuntimePanel` will exist but I'll check that `CreatePanel` won't